src/fs/dvfs/dvfs.h

```diff
@@ -28,6 +28,7 @@
 #define DVFS_DIR_VIRTUAL   0x01000000
 #define DVFS_CHILD_VIRTUAL 0x02000000
 #define DVFS_MOUNT_POINT   0x04000000
+#define DVFS_NO_LSEEK      0x08000000
```

Comment:
It looks not very good

Reply:
Why? Macro name looks pretty self-explanatory.

Reply:
It's not a node flag, better if it's be a driver specific, in my mind

Reply:
It's only for the future

Reply:
When this flag is used in vfs layer, there's no need to implement stubs for `truncate` calls when this calls have no sense (for example, for char devices).